src/site/content/en/blog/carousel-best-practices/index.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: Best practices for carousels
+subhead: Optimize carousels for performance and usability.
+authors:
+  - katiehempenius
+description:  Learn how to optimize carousels for performance and usability.
+date: 2021-01-26
+hero: hero.jpeg
```

Comment:
Please try lightly optimizing the hero image to keep the repo size down :) https://web.dev/handbook/markup-media/#optimization